src/SprykerSdk/Acp/Console/AppTranslationCreateConsole.php

```diff
@@ -29,19 +35,56 @@ class AppTranslationCreateConsole extends AbstractConsole
      */
     public const TRANSLATION_FILE = 'translation-file';
 
+    /**
+     * @var string
+     */
+    public const MANIFEST_FOLDER = 'manifest-folder';
+
+    /**
+     * @var string
+     */
+    public const CONFIGURATION_FILE = 'configuration-file';
+
     /**
      * @var string
      */
     public const TRANSLATION_FILE_SHORT = 't';
 
+    /**
+     * @var string
+     */
+    public const MANIFEST_FOLDER_SHORT = 'm';
+
+    /**
+     * @var string
+     */
+    public const CONFIGURATION_FILE_SHORT = 'c';
+
+    /**
+     * @var string
+     */
+    protected const CHOICE_NEW_LOCALE = 'Select this to add a new locale';
+
+    /**
+     * @var \Symfony\Component\Console\Input\InputInterface
+     */
+    protected InputInterface $input;
```

Comment:
we don't use property type hints so far
please, check all places

Reply:
Why not? There is no rule against it.

Reply:
There was a message from Mark Scherer in Slack, that we are not ready to use them yet, because in this case, we need to initialize these properties when we create an instance of a class and need to investigate all proc and cons using the typed properties in Spryker

Reply:
https://spryker.slack.com/archives/C03KVH4N8/p1634553649381900

Reply:
as you wish